frontend/src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { registerRootComponent } from 'expo';
+import { View, Text } from 'react-native';
+import Troop from './Troop';
+
+const style = {
+  flex: 1,
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: '#000',
+};
+
+function App() {
+  return (
+    <View style={style}>
+      <Troop/>
+    </View>
+  );
+}
```

Comment:
you can import CSS styles since you have created an extra styles.js file.
```js
import commonStyles from '../common/styles';

function App() {
  return (
    <View style={style={commonStyles.container}}>
      <Troop/>
    </View>
  );
}

```

Reply:
fixed now :)

> Some pages need new line at the bottom

Fixed now